### PR TITLE
Site updates

### PR DESCRIPTION
**Pull request by michaeltaranto**
Some tweaks to the site.

### CODE REVIEW

File: site/docs/setup.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ require('esbuild')
 
 ## Vite
 
-> Warning: Currently the Vite plugin doesn't rebuild files when dependent files change, e.g. updating `theme.css.ts` should rebuild `styles.css.ts` which imports `theme.css.ts`. This is a limitation in the Vite Plugin API that will hopefully be resolved soon. You can track the Vite issue here: https://github.com/vitejs/vite/issues/3216
+> Warning: Currently the Vite plugin doesn't rebuild files when dependent files change, e.g. updating `theme.css.ts` should rebuild `styles.css.ts` which imports `theme.css.ts`. This is a limitation in the Vite Plugin API that will hopefully be resolved soon. You can track the Vite issue [here](https://github.com/vitejs/vite/issues/3216).
```
Comment (markdalgleish) [inline site/docs/setup.md:98]: How about this?

```suggestion
> Warning: Currently the Vite plugin doesn't rebuild files when dependent files change, e.g. updating `theme.css.ts` should rebuild `styles.css.ts` which imports `theme.css.ts`. This is a [limitation in the Vite Plugin API](https://github.com/vitejs/vite/issues/3216) that will hopefully be resolved soon.
```

File: .github/workflows/validate.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ jobs:
         run: yarn --immutable
 
       - name: Build
-        run: yarn build
+        run: yarn build && node ./site/makeDocsManifest
```
Comment (markdalgleish) [inline .github/workflows/validate.yml:27]: Is this redundant now? Looks like it's been added to the build script.

File: site/src/system/styles/reset.css.ts [hunk 3]
```diff
@@ -14,6 +14,7 @@ export const base = style({
 export const button = style({
   background: 0,
   border: 0,
+  WebkitTapHighlightColor: 'rgba(0, 0, 0, 0)',
```
Comment (markdalgleish) [inline site/src/system/styles/reset.css.ts:17]: 👏

**Review by mattcompiles — APPROVED** [review]
Looks good.

**Pull request merged**